frontend/src/container/Header.jsx

```diff
@@ -120,6 +124,16 @@ class Header extends React.Component {
                 <MenuActionListAdminLink t={this.props.t} />
               }
 
+              <li className='header__menu__rightside__calendar'>
+                <button
+                  className='btn outlineTextBtn primaryColorBorder nohover'
+                  onClick={this.handleClickCalendar}
```

Comment:
handleClickCalendar => handleClickCalendarButton

Reply:
to fix

Reply:
renamed